src/web/backend/nextcloud/models.py

```diff
@@ -0,0 +1,21 @@
+""" Models to work with ldap objects, operated by EDAP library """
+
+
+class User:
+    """ Edap user model """
+    def __init__(self, fqdn, uid, given_name, mail, surname, object_class=None):
+        self.fqdn = fqdn
```

Comment:
`fqdn` depends on the chosen LDAP schema. Therefore, one should deduce this from EDAP or app configuration, and it shouldn't be considered as a generic `User` attibute.

Reply:
I understand, but for me it makes sense to move it to super class like `BaseLdapModel`, to not construct it every time we need it, because some methods need `fqdn` of the object. What do you think about it ?

Reply:
@matejak this sounds reasonable, but we will need similar models like User for every object (I already implemented Division, Franchise models in #19) and all of them have fqdn, is it ok to repeat same for them `LdapDivision(Division)`, `LdapFranchise(Franchise)`?

Reply:
I don't like that a private interface creeps to another layer, but I don't want to make it too academic either. Therefore, do what makes sense to you most. It is a design decision of lower importance, and it may turn out that my approach is not even better.

Reply:
> I don't like that a private interface creeps to another layer
Now for me it also looks worse, changed it to User -> LdapUser, thanks